portal/cookbook_gallery.yaml

```diff
@@ -1,7 +1,7 @@
-- title: CMIP6
-  url: http://gallery.pangeo.io/repos/pangeo-gallery/cmip6/
+- title: Radar Cookbook
+  url: https://projectpythiatutorials.github.io/radar-cookbook/landing-page.html
   description: |
-    Examples of analysis of Google Cloud CMIP6 data using Pangeo tools.
+    This Project Pythia Cookbook covers the basics of working with weather radar data in Python using Py-ART.
```

Comment:
Can you remove the mention of Py-ART here? Although the current content only covers that material, we want to make this flexible enough to add other Python modules (Ex. wradlab) in the future

Reply:
Done! I was a little curious about that. I've also added tags in this push.